### PR TITLE
Update serverless-business-scenarios.md

### LINKED ISSUE #16815: Broken links
@JeremyLikness there are two broken links in this content and I'm not sure if there are good replacements for them

https://azure.microsoft.com/resources/samples/functions-dotnet-photo-mosaic/
https://azure.microsoft.com/resources/samples/iot-hub-node-ping/

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fef858f1-a7a7-3a73-4aff-b9654fc60396
* Version Independent ID: b800be23-e314-ae9b-b0b3-cdced7e6121c
* Content: [Sample business scenarios and use cases for serverless apps](https://docs.microsoft.com/en-us/dotnet/architecture/serverless/serverless-business-scenarios)
* Content Source: [docs/architecture/serverless/serverless-business-scenarios.md](https://github.com/dotnet/docs/blob/master/docs/architecture/serverless/serverless-business-scenarios.md)
* Product: **dotnet-architecture**
* Technology: **serverless**
* GitHub Login: @JeremyLikness
* Microsoft Alias: **jeliknes**

### PR DESCRIPTION
**Pull request by JeremyLikness**
Update links that have moved from Azure Samples. This addresses issue #16815. /cc @mairaw

## Summary

Updated links to samples to proper repositories.

Fixes #16815 (if available)

### CODE REVIEW

File: docs/architecture/serverless/serverless-business-scenarios.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The sample uses Azure Functions and Microsoft Cognitive Services Custom Vision S
 
 For example, you can train your model with Orlando landmarks, such as the Orlando Eye. Custom Vision will recognize an image of the Orlando Eye, and the function will create a photo mosaic composed of Bing image search results for "Orlando Eye."
 
-For more information, see [Azure Functions photo mosaic generator](https://azure.microsoft.com/resources/samples/functions-dotnet-photo-mosaic/).
+For more information, see [Azure Functions photo mosaic generator](https://github.com/lindydonna/photo-mosaic).
```
Comment (mairaw) [inline docs/architecture/serverless/serverless-business-scenarios.md]: should we link here instead?

```suggestion
For more information, see [Azure Functions photo mosaic generator](https://github.com/Azure-Samples/functions-dotnet-photo-mosaic).
```

File: docs/architecture/serverless/serverless-business-scenarios.md [hunk 2]
```diff
@@ -77,12 +77,12 @@ The full link shortener repository with instructions is available here: [Serverl
...
 The sample consists of an Azure IoT Hub and an Azure Function. A new message on the IoT Hub triggers the Azure Function. The serverless code sends the same message content back to the device that sent it. The project has all the code and deployment configuration needed for the solution.
 
-For more information, see [Azure IoT Hub ping](https://azure.microsoft.com/resources/samples/iot-hub-node-ping/).
+For more information, see [Azure IoT Hub ping](https://github.com/Azure-Samples/iot-hub-node-ping).
 
 ## Recommended resources
 
-- [Azure Functions photo mosaic generator](https://azure.microsoft.com/resources/samples/functions-dotnet-photo-mosaic/)
-- [Azure IoT Hub ping](https://azure.microsoft.com/resources/samples/iot-hub-node-ping/)
+- [Azure Functions photo mosaic generator](https://github.com/lindydonna/photo-mosaic)
```
Comment (mairaw) [inline docs/architecture/serverless/serverless-business-scenarios.md]:
```suggestion
- [Azure Functions photo mosaic generator](https://github.com/Azure-Samples/functions-dotnet-photo-mosaic)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @JeremyLikness for the prompt response to my issue. Can you just check if my suggestions make sense?

**Comment by Youssef1313** [comment]
I had a PR open already #16827 😄 
Should I close it ?

**Comment by mairaw** [comment]
I see that @Youssef1313 also opened a PR to fix these and other links. Should we close this in favor of #16827?

**Comment by mairaw** [comment]
jinx 😄

**Comment by JeremyLikness** [comment]
Yes, the other fix is better and touches on additional links I did not address. Sorry for the redundant work but @mairaw @Youssef1313 the #16827 looks good to me.

**Pull request closed without merging**